src/sentry/models/dsymfile.py

```diff
@@ -29,7 +29,7 @@
     SymCache, SYMCACHE_LATEST_VERSION
 
 from sentry import options
-from sentry.utils.cache import cache
```

Comment:
What's the reason for changing the caching here? It's changing from `memcached` to `redis` with this. Is there a reason this needs to change to redis?

I'm not blocking this because I think it's wrong, I just want to make sure this was a conscious decision and understand why.

Reply:
Taking a stab at guessing, it seems that the data isn't truly ephemeral? Is it because we actually require the keys to exist for the pipeline to work?

Reply:
I think the reason was to get it working in the dev environment out of the box. That cached value is needed to complete a successful dsym upload.

Reply:
Yup, was just making sure that was the reason. If so, that makes sense. Thanks. :)

Reply:
Ah, I should've read your comment more carefully. It's not the pipeline that requires that value, but the CLI depends on it. `sentry-cli upload-*` polls that `state` field, waits until all symbols have been processed and then displays success/error info to the user. While the success case is trivial (we create a projectdsymfile and query that), the error case is a little harder. It doesn't really make sense to persist anything in the database and then start GC'ing it as it only has to stick around while other symbols from the same CLI upload are still processing (and thus the CLI is still polling).

Reply:
Nah, that's what I meant. "pipeline" was a very general term here. Not our event processing pipeline, but rather, some other series of events that are happening that need coordination, as you confirmed.